tests/manage/pv_services/test_resource_deletion_during_pvc_pod_deletion_and_io.py

```diff
@@ -68,28 +68,41 @@ def setup_base(
         """
         Create PVCs and pods
         """
+        access_modes = [constants.ACCESS_MODE_RWO]
+        if interface == constants.CEPHFILESYSTEM:
+            access_modes.append(constants.ACCESS_MODE_RWX)
         pvc_objs = multi_pvc_factory(
             interface=interface,
             project=None,
             storageclass=None,
             size=self.pvc_size,
-            access_mode=constants.ACCESS_MODE_RWO,
+            access_modes=access_modes,
+            access_modes_selection='distribute_random',
```

Comment:
The idea of having a random distribution of PVC access modes is great but in a context of regression testing, we could get different results between executions because different sets of PVCs were created

Reply:
access_modes_selection='distribute_random' is not for performing random selection of access mode. That option will distribute access modes among the PVCs. Just the order creation of PVCs will be independent of the access mode. It will always create same number of RWO and RWX PVC.

Reply:
ok, thanks for the explanation